### PR TITLE
Readability review

### PR DESCRIPTION
**Pull request by andrewsg**
Style improvements and rework to use explicit imports.

R: @filbranden @mingzhaodotname CC: @isdal

### CODE REVIEW

File: multicore_runtime/wsgi_test.py [hunk 1]
```diff
@@ -21,36 +20,50 @@
...
 from google.appengine.api import appinfo
 
+def script_path(script, test_name=__name__):
+  """Returns a fully qualified module path based on test_name."""
+  return '{test_path}.{script}'.format(
+      test_path=test_name, script=script)
+
+def static_path(relative_path, test_path=os.path.dirname(__file__)):
+  """Returns a fully qualified static file path based on test_path."""
+  return '{base}{slash}{relative_path}'.format(
```
Comment (filbranden) [inline multicore_runtime/wsgi_test.py]: Sorry I missed this before... but how is this different from `os.path.join(test_path, relative_path)`?

File: multicore_runtime/wsgi_test.py [hunk 2]
```diff
@@ -77,55 +90,55 @@
...
-  for name in sorted(os.environ):
-    resp += '%s=%s\n' % (name, os.environ[name])
-  return Response(resp)
+  return wrappers.Response(''.join(
+      '%s=%s\n' % (k, v) for k, v in sorted(os.environ.iteritems())
+      ))
 
 
 class MetaAppTestCase(unittest.TestCase):
+  # pylint: disable=g-import-not-at-top
```
Comment (filbranden) [inline multicore_runtime/wsgi_test.py]: I believe imports are only expected in `setUp` so you could perhaps move this there, or perhaps even only on the two import lines (`import vmconfig` and `import wsgi`).

File: multicore_runtime/wsgi_test.py [hunk 3]
```diff
@@ -77,55 +90,55 @@
...
 
   def setUp(self):
     # Pre-import modules to patch them in advance.
     from google.appengine.ext.vmruntime import vmconfig  # pylint: disable=unused-variable
```
Comment (filbranden) [inline multicore_runtime/wsgi_test.py]: I believe the `pylint: disable=unused-variable` is no longer needed, now that you're using `patch.object(vmconfig, ...)` this symbol is now in use.

**Comment by filbranden** [comment]
Three last minute nitpicks (sorry about only seeing those now), otherwise looks good!

@mingzhaodotname Feel free to merge this PR once you're happy with it.

Cheers,
Filipe

**Comment by andrewsg** [comment]
Fixed the above nits and squashed (so it's ready to be merged as-is if Ming approves).

**Comment by filbranden** [comment]
Looks great, thanks!

**Comment by mingzhaodotname** [comment]
LGTM

**Comment by filbranden** [comment]
Hi @mingzhaodotname, would you merge this CL? Or are you going to wait until @andrewsg is back next week for him to merge it himself?

Cheers,
Filipe

**Comment by mingzhaodotname** [comment]
I think I will wait for @andrewsg <https://github.com/andrewsg> to merge it
:-)

**Pull request merged**